File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/ContentAuditVO.kt
package com.sdkwork.app

data class ContentAuditVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val taskId: String? = null,
    val contentId: String? = null,
    val overallResult: String? = null,
    val riskLevel: String? = null,
    val results: Map<String, AuditResultItem>? = null,
    val textResult: TextAuditVO? = null,
    val imageResults: List<ImageAuditVO>? = null,
    val videoResult: VideoAuditVO? = null,
    val audioResult: AudioAuditVO? = null,
    val suggestion: String? = null
)

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/content_audit_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ContentAuditVO:
    """综合内容审核响应"""
    created_at: str = None
    updated_at: str = None
    task_id: str = None
    content_id: str = None
    overall_result: str = None
    risk_level: str = None
    results: Dict[str, AuditResultItem] = None
    text_result: TextAuditVO = None
    image_results: List[ImageAuditVO] = None
    video_result: VideoAuditVO = None
    audio_result: AudioAuditVO = None
    suggestion: str = None
